examples/replay/config-0.25-degree.yaml

```
@@ -211,3 +211,79 @@ FV3Dataset:
         - tmp
         - ugrd
         - vgrd
+
+MOM6Dataset:
+    file_prefixes:
+        - ocn_
+    
+    path_out: "gcs://noaa-ufs-gefsv13replay/ufs-hr1/0.25-degree/03h-freq/zarr/"
+    coords_path_out: "gcs://noaa-ufs-gefsv13replay/ufs-hr1/0.25-degree/coordinates/zarr/"
+    forecast_hours: [0]
```

I was wrong, I'm glad we checked with Phil. The following would be consistent with the atmosphere data that's already online
```suggestion
    forecast_hours: [-6]
```